dpdata/lammps/dump.py

```
@@ -197,7 +197,91 @@ def load_file(fname: FileType, begin=0, step=1):
+        Path to the input file.
+
+    Returns
+    -------
+    list or None
+        List of spin info keys if found, None otherwise.
+    """
+    if inputfile is None:
+        return None
+
+    if not os.path.isfile(inputfile):
+        warnings.warn(f"Input file {inputfile} not found.")
```

Ruff

No explicit `stacklevel` keyword argument found

(B028)